<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>578</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100000</width>
     <height>100000</height>
    </size>
   </property>
   <widget class="QWidget" name="codigo" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>361</width>
      <height>501</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>361</width>
      <height>501</height>
     </size>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>351</width>
       <height>491</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="lineWrapMode">
         <enum>QPlainTextEdit::NoWrap</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QCommandLinkButton" name="CompileButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Compile</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>399</x>
      <y>69</y>
      <width>561</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>579</y>
      <width>931</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPlainTextEdit" name="terminal">
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>263</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btn_start">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_reset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_pause">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_New_File"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionParser_Tree"/>
   </widget>
   <widget class="QMenu" name="menuTurtle">
    <property name="title">
     <string>Turtle</string>
    </property>
    <addaction name="actionShow_Grid"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTurtle"/>
  </widget>
  <action name="action_New_File">
   <property name="text">
    <string>New File</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionParser_Tree">
   <property name="text">
    <string>Parser Tree</string>
   </property>
  </action>
  <action name="actionShow_Grid">
   <property name="text">
    <string>Show Grid</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
